src-tauri/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "phosphobot-desktop",
+  "version": "0.3.56",
```

Comment:
same question: does this package number needs to be kept up to date? 
how can we manage version numbers more easily? can this file be a template filled in with the version number at compile time ?